### PR TITLE
Bump to version 0.0.3

### PR DESCRIPTION
**Pull request by Anemy**
First updates to the `changelog.md` following https://keepachangelog.com/en/1.0.0/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,36 @@ All notable changes to the "mongodb" extension will be documented in this file.
...
+
+### Changed
+
+- README copy tweaks.
+
+## [0.0.2] - 2020-05-13
+
+### Changed
+
+- Marketplace preview release
```
Comment (alenakhineika) [inline CHANGELOG.md:28]: Do we do dots at the end? Some release notes with and some without.

Comment (Anemy) [reply CHANGELOG.md:28]: Any preference? I feel sometimes it looks better with sometimes without lol

Comment (alenakhineika) [reply CHANGELOG.md:28]: I prefer when it is consistent, all with or all without :)

Comment (Anemy) [reply CHANGELOG.md:28]: Sound good - just pushed up all without

**Pull request merged**